server/pages/index.js

```diff
@@ -171,13 +171,12 @@ const getFilesForChunk = ( chunkName, request ) => {
 	return groupAssetsByType( allTheFiles );
 };
 
-const getFilesForEntrypoint = target => {
-	const entrypointAssets = getAssets( target ).entrypoints.build.assets.filter(
+const getFilesForEntrypoint = ( target, name ) => {
```

Comment:
This appears to take into account the changes in the multiple builds PR correctly 👍